logever/src/main/java/ai/smartfac/logever/entity/User.java

```diff
@@ -45,11 +47,16 @@ public class User implements UserDetails {
     private Timestamp createDt;
     @Column(name = "updated_by")
     @LastModifiedBy
+    @JsonIgnore
     private String updatedBy;
     @JsonIgnore
     @Column(name="update_dt")
     @UpdateTimestamp
     private Timestamp updateDt;
```

Comment:
We will also need status of the user, ACTIVE/INACTIVE

Reply:
we have isActive.